authentication/views.py

```diff
@@ -84,6 +85,20 @@ def get_serializer_cls(self):
         """Return the serializer cls"""
         return RegisterEmailSerializer
 
+    def post(self, request):
+        """ Verify recaptcha response before proceeding """
+        if settings.RECAPTCHA_SITE_KEY:
+            r = requests.post(
+                "https://www.google.com/recaptcha/api/siteverify?secret={}&response={}".format(
+                    settings.RECAPTCHA_SECRET_KEY,
+                    request.data["recaptcha"]
```

Comment:
Can we also make these named parameters just for clarity?

Reply:
done